lisa/util/process.py

```diff
@@ -77,13 +77,13 @@ def start(
             if not self._is_posix:
                 split_command = ["cmd", "/c", command]
             elif sudo:
-                command = f"sudo -s {command}"
+                command = f"sudo -s sh -c '{command}'"
                 split_command = shlex.split(command, posix=self._is_posix)
             else:
                 split_command = ["sh", "-c", command]
         else:
             if sudo and self._is_posix:
-                command = f"sudo {command}"
+                command = f"sudo sh -c '{command}'"
```

Comment:
this means sudo without shell. If add it, there is no way to disable shell with sudo.